src/Symfony/Component/Translation/Translator.php

```diff
@@ -491,4 +492,19 @@ protected function assertValidLocale($locale)
             throw new \InvalidArgumentException(sprintf('Invalid "%s" locale.', $locale));
         }
     }
+
+    /**
+     * @see WarmableInterface
+     *
+     * @param string $cacheDir only used for interface compatibility, $this->cacheDir is already defined
+     */
+    public function warmUp($cacheDir)
+    {
+        if ($this->locale !== null) {
+            $this->loadCatalogue($this->locale);
+        }
+        foreach ($this->fallbackLocales as $locale) {
```

Comment:
Actually, that won't do the trick. If I try to load ```cz``` catalogue and it's not a registered locale, I'll have a ```cz``` catalogue with fallback translations (i.e ```fr```) computed in.
It won't generate catalogues for every fallback. Am I right ?